faq/account.mdx

```diff
@@ -0,0 +1,160 @@
+
+</Concept>
+
+<Concept>
+
+## Can I change my Organization’s country?
+
+If you need to change your Organization country and region, you must [create a support ticket](https://console.scaleway.com/support/tickets/create).
+
+To make the process faster, when creating your ticket, inform us your Organization’s new address, including: street address, city, postal code, region, and country.
+
+</Concept>
+
+<Concept>
+
+## Can I move a Project to another account?
+
+At this moment, it is not possible to move a Project from one account to another.
+
+</Concept>
+
+<Concept>
+
+## Can I recover a deleted account?
+
+No, it is not possible to recover a deleted account.
+
+If you only want to pause your account and keep your existing resources, you can [deactivate it](/console/account/how-to/close-account/#how-to-deactivate-your-organization) instead.
+
+</Concept>
+
+<Concept>
+
+## How do I change my email address?
+
+To change your email address:
+
+1. On the [Scaleway console](https://console.scaleway.com), go to your [profile](https://console.scaleway.com/account/profile).
+2. Under the section **User information**, click **Edit profile**.
+3. Enter a new address under **Email address**, then click **Confirm changes** to save your modification.
+
+Your email address is now updated.
+
+</Concept>
+
+<Concept>
+
+## How can I get more quotas?
+
+You can get more quotas by adding and verifying your [payment method](/console/my-account/how-to/manage-billing-information/) and [identity](/console/my-account/how-to/verify-identity/).
+
+If you have already completed these two steps and still need an increase in quotas, contact our [support team](https://console.scaleway.com/support/tickets/create) and inform them which product you need more quotas for.
+
+</Concept>
+
+<Concept>
+
+## I don’t seem to have permission to my Organization's resources
+
+If you do not have access to certain Organization resources, you might not have the necessary [IAM](/identity-and-access-management/iam/concepts/#iam) **permission set**.
+
+[Read the documentation on permission sets](/identity-and-access-management/iam/reference-content/permission-sets/)​ to learn more about the different access levels.
+
+Contact an Organization owner or a member with the necessary permissions to request rights to the resources you require access to.
+
+</Concept>
+
+<Concept>
+
+## What is the difference between deactivating and deleting an account?
+
+When your account is deactivated, we preserve your data and settings, making it possible to recover it later if you wish. Note that deactivated accounts are not billed.
+
+On the other hand, when you delete an account, we permanently erase the existing resources and any data and settings. It is not possible to recover a deleted account.
+ 
+</Concept>
+
+<Concept>
+
+## How do I deactivate my Organization?
+
+Deactivating an Organization **permanently deletes** all your services and backups.
```

Comment:
This seems to go against what was previously said - that when accounts were deactivated data and settings and resources are preserved?

"If you only want to pause your account and keep your existing resources, you can [deactivate it](/console/account/how-to/close-account/#how-to-deactivate-your-organization) instead."

"When your account is deactivated, we preserve your data and settings, making it possible to recover it later if you wish."

Reply:
Thank you, Ro. You're right. 
I'm trying to confirm the precise conditions with Excellence to update the text. 🙏🏾